formbuilder/styles.css

```diff
@@ -132,6 +132,10 @@ textarea.json-viewer {
   height: 100px;
 }
 
+.edit-in-place:hover {
```

Comment:
This target the form title and description whenever it's in "edit mode" or not.
Do we want cursor pointer when theses fields became input text ?
If not, use ```span.edit-in-place:hover```